### PR TITLE
network: about Gateway=_dhcp4 or _ipv6ra

### PR DESCRIPTION
**Pull request by yuwata**

### CODE REVIEW

File: src/network/networkd-route.c [hunk 1]
```diff
@@ -1743,7 +1743,7 @@ int config_parse_gateway(
                         return 0;
                 }
 
-                if (streq(rvalue, "_dhcp6")) {
+                if (streq(rvalue, "_ra")) {
```
Comment (poettering) [inline src/network/networkd-route.c]: hmm, this is compat breakage, no? Or did we never release with this change?

maybe call this `_ipv6ra`?

(dunno, just wondering)

Comment (poettering) [reply src/network/networkd-route.c]: ah, i see now, it was added in d306d1d0ca2e6559a598a6eb459f65f9377f909e only, which we haven#t released so far.

**Review by poettering — APPROVED** [review]
one minor comment, lgtm

**Comment by poettering** [comment]
The rename suggestions for _ra → _ipv6ra is just an idea. Either works for me, hence setting green label

**Comment by yuwata** [comment]
@poettering Thank you for the review. Now `_ra` is renamed to `_ipv6ra`. Setting the green label.

**Comment by kpfleming** [comment]
It's almost five years later, but still... thank you for adding this! Just today I had a need to add some routes which have to go to the default IPv6 gateway (because otherwise they appear to be directly connected), and didn't know how I was going to avoid including the gateway's address in the configuration file. `_ipv6ra` to the rescue, works perfectly!

**Comment by yuwata** [comment]
@kpfleming Happy to hear that.

**Pull request merged**